### ISSUE TITLE
Bug:  nn.MaxUnpool1d example throws error

### ISSUE THREAD

**Issue by ZhuBaohe** [pytorch/pytorch#17004]
I found some error when running nn.MaxUnpool1d example in  [doc](https://pytorch.org/docs/stable/nn.html?highlight=maxunpool1d#torch.nn.MaxUnpool1d).
```
import torch 
import torch.nn as nn 

pool = nn.MaxPool1d(2, stride=2, return_indices=True)
unpool = nn.MaxUnpool1d(2, stride=2) 
 
# Example showcasing the use of output_size
input = torch.tensor([[[1., 2, 3, 4, 5, 6, 7, 8, 9]]])
output, indices = pool(input)
unpool(output, indices, output_size=input.size())
```
The output is shown below.
![1](https://user-images.githubusercontent.com/35796307/52639687-c3f2e080-2f0f-11e9-8f47-e1fdd3917217.jpg)

**Comment by vishwakftw** [comment]
This is a duplicate of https://github.com/pytorch/pytorch/issues/16486, and has been fixed in PyTorch version 1.0.1. I suggest that you update your installation to reflect this change.

**Comment by SsnL** [comment]
see @vishwakftw 's reply
